elastic-buffer-sim/src/Bittide/Topology/TH.hs

```diff
@@ -366,7 +368,7 @@ extractPeriods _ = pure (Clash.snatToNum (Clash.clockPeriod @dom))
 -- | Given a graph with \(n\) nodes, generate a function which takes a list of \(n\)
 -- offsets (divergence from spec) and returns a tuple of signals for each clock
 -- domain
-simNodesFromGraph :: ClockControlConfig -> Graph -> Q Exp
+simNodesFromGraph :: ClockControlConfig m -> Graph -> Q Exp
```

Comment:
`n` nodes, or `m`, nodes?

Reply:
Neither, it should be `nDataCountBits` :-). I.e., it sets the FIFO size.